example/docs/components/CircleStencil.mdx

```diff
@@ -5,6 +5,8 @@ sidebar_position: 3
 
 import TOCInline from "@theme/TOCInline";
 
+Used to display the current cropped area and change it by interaction with it.
```

Comment:
Simple high level details when landing on the components pages might help new users with the learning curve. Details source from existing content in `components-and-hooks.mdx`